lib/master_proxy/cowboy2_handler.ex

```diff
@@ -41,7 +41,7 @@ defmodule MasterProxy.Cowboy2Handler do
 
   defp dispatch(%{phoenix_endpoint: endpoint}, req) do
     # we don't pass in any opts here because that is how phoenix does it
-    # see https://github.com/phoenixframework/phoenix/blob/master/lib/phoenix/endpoint/cowboy2_adapter.ex#L42
```

Comment:
Do you mind changing this to a permalink? I worry sometimes that pointing to master will mess up the line numbers as new commits come in.

Reply:
Well this is changing the link away from master for exactly that reason. Would you prefer a direct sha instead of a tag? Generally I would trust the phoenix team to not update existing tags, but I'd be glad to change it to a sha instead if you'd prefer that.

Reply:
Oh haha, sorry about that. I read the change backwards!